### PR TITLE
RELEASE_NOTES: Update for 0.8.0

### PR DESCRIPTION
**Pull request by hypothesist**

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+  errors using ``happiness``.
+
+- ``WriteGoodLintBear`` - Lints the text files using ``write-good`` for
+  improving proses.
+
+- ``coalaBear`` - Check for the correct spelling and casing of ``coala``
+  in the text files.
+
+- ``VultureBear`` - Checks Python code for unused variables and functions
+  using ``vulture``.
```
Comment (SanketDG) [inline RELEASE_NOTES.rst]: could you link to all of the above tools if its not a big deal? :)

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+  and just the necessary bears (and their dependencies) can be installed.
+
+- Also as a part of the decentralization process, several bears now have
+  the ``REQUIREMENTS`` attribute. This is one of the requirement objects
+  supporting various package managers such as ``apt-get``, ``dnf``,
+  ``yum``, ``pip``, ``npm``, ``gem``, and so on. To learn more, please
+  see the ``coala`` 0.8.0 release changelog.
+
+- Several bears now support the ``ASCIINEMA_URL`` attribute. This contains
+  an URL to an asciinema video displaying the bear's working.
```
Comment (sils1297) [inline RELEASE_NOTES.rst]: those don't really belong to the bears changelog do they?

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+
+**Bug Fixes**
+
+- An issue in ``FilenameBear`` involving files with fully capitalized names
+  has been resolved. `Pull #687 <https://github.com/coala-analyzer/coala-bears/pull/687>`_
+
+- Various tiny issues with ``InvalidLinkBear`` involving some false positives
```
Comment (sils1297) [inline RELEASE_NOTES.rst]: s/tiny issues/corner cases/

File: RELEASE_NOTES.rst [hunk 4]
```diff
@@ -1,3 +1,68 @@
+coala-bears 0.8.0
+=================
+
+For this release, we have had 19 contributors from around the world
+contributing about 176 commits to just coala-bears in the past 9 weeks.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: about?..

File: RELEASE_NOTES.rst [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+
+Here are the important changes or additions:
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: more like ``and additions``

File: RELEASE_NOTES.rst [hunk 6]
```diff
@@ -1,3 +1,68 @@
...
+**New Bears**
+
+- ``CSecurityBear`` - Lints C/C++ files and identifies possible security
+  issues.
+
+- ``HappinessLintBear`` - Checks JavaScript files for semantic and syntax
+  errors using ``happiness``.
+
+- ``WriteGoodLintBear`` - Lints the text files using ``write-good`` for
+  improving proses.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: for improved proses* ? or ``to improve proses`` ? not sure

File: RELEASE_NOTES.rst [hunk 7]
```diff
@@ -1,3 +1,68 @@
...
+
+- Bear upload tool - this is a part of the complete decentralization of
+  bears. With this tool, bears are uploaded as individual packages to PyPI
+  and just the necessary bears (and their dependencies) can be installed.
+
+- Also as a part of the decentralization process, several bears now have
+  the ``REQUIREMENTS`` attribute. This is one of the requirement objects
+  supporting various package managers such as ``apt-get``, ``dnf``,
+  ``yum``, ``pip``, ``npm``, ``gem``, and so on. To learn more, please
+  see the ``coala`` 0.8.0 release changelog.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: link to it maybe :D ?

File: RELEASE_NOTES.rst [hunk 8]
```diff
@@ -1,3 +1,68 @@
...
+  `Issue #691 <https://github.com/coala-analyzer/coala-bears/issues/691>`_
+
+**Documentation**
+
+- A complete overhaul of the README page with a special emphasis on design
+  and user-friendliness.
+
+**Regressions**
+
+- Python 3.3 support is dropped.
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: as you used all in past, use "was"

File: RELEASE_NOTES.rst [hunk 9]
```diff
@@ -1,3 +1,68 @@
...
+- ``CSecurityBear`` - Lints C/C++ files and identifies possible security
+  issues.
+
+- ``HappinessLintBear`` - Checks JavaScript files for semantic and syntax
+  errors using ``happiness``.
+
+- ``WriteGoodLintBear`` - Lints the text files using ``write-good`` for
+  improving proses.
+
+- ``coalaBear`` - Check for the correct spelling and casing of ``coala``
```
Comment (Asnelchristian) [inline RELEASE_NOTES.rst]: ``Checks``

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (Asnelchristian) [inline RELEASE_NOTES.rst]: you forgot about the ``YapfBear``

Comment (sils1297) [inline RELEASE_NOTES.rst]: @hypothesist just to confirm, did you go through the commit log? It'd be shame to forget something here with all the great stuff that happened.

File: RELEASE_NOTES.rst (hunk 3 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: did you just speak VIML to me? I like you :P

File: RELEASE_NOTES.rst (hunk 4 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: about stuff

just kidding. I'll make up a brief description :)

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: Well, they have the asciinema link which is specific to each bear. I think that should go in to the changelog.

Or do you mean the second sentence  of that point?

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: @SanketDG No problem :+1:

@Asnelchristian sorry about that, adding it in right away

@sils1297 Went through each commit. No idea how I missed YapfBear. Unless it was camouflaged between Asnel's commits. I'll do the whole list once more and get back to you.

File: RELEASE_NOTES.rst (hunk 6 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: `to improve proses` sounds good :+1:

Comment (hypothesist) [inline RELEASE_NOTES.rst]: BTW those lines are just copied from the bear's annotation description :P

File: RELEASE_NOTES.rst (hunk 3 above)
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: no idea what viml is, but that looks like``sed``

File: RELEASE_NOTES.rst (hunk 4 above)
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: nono, what i meant is that you used "about 176 commits". how acn you say "about" and then such a precise number?

File: RELEASE_NOTES.rst (hunk 3 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: Oh VIML is vim's programming language

But I guess Lasse meant sed

File: RELEASE_NOTES.rst (hunk 4 above)
Comment (hypothesist) [inline RELEASE_NOTES.rst]: It's exactly 176 commits :P

I'll remove the about part

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala-bears/pull/702/commits/4b6331a196cf05c243e9103877b51f6604ce2caa is ready

**Comment by hypothesist** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/702/commits/f7f9beb46459db2cd5eb787265750396a4653b22

**Comment by hypothesist** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/702/commits/6b2b906c96ca3448cc758b22da4e50a0e9a23344

**Comment by hypothesist** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/702/commits/0dc33e1f4d42370a45d3038f7b37e49737e3d39c

**Comment by hypothesist** [comment]
@rultor merge

**Pull request merged**